android/build.gradle

```
@@ -110,16 +137,37 @@ def shouldAssertNoMultipleInstances() {
   }
 }
 
-def noMultipleInstancesAssertion() {
-    // Assert there are no multiple installations of Reanimated
-    Set<File> files = fileTree(rootDir.parent) {
+def findReanimatedInstancesForPath(String path) {
+    return fileTree(path) {
         include "**/react-native-reanimated/package.json"
         exclude "**/.yarn/**"
+        exclude {{ file, attr -> attr.isSymbolicLink() }}
     }.files
+}
 
+def noMultipleInstancesAssertion() {
```

Let's make it analogous to iOS.
```suggestion
def assertNoMultipleInstances() {
```